long_cat_is_long.py

```diff
@@ -0,0 +1,35 @@
+import string
+from collections.abc import Generator
+
+
+def words(text: str) -> Generator[str]:
+    """
+    This is generator receive text, split it to the word without the punctuations.
+    :param text: The text.
+    :return: The words without punctuations.
+    """
+    text = text.split()
+    for word in text:
+        for character in string.punctuation:
```

Comment:
There are more elegant and efficient ways of removing punctuation from strings. 
https://datagy.io/python-remove-punctuation-from-string/#:~:text=One%20of%20the%20easiest%20ways,translate()%20method.